vowpalwabbit/example_predict.h

```diff
@@ -62,3 +63,7 @@ class VW_DEPRECATED("example_predict is now RAII based. That class can be used i
 
   void clear();
 };
+
+std::string features_to_string(const example_predict& ec);
+std::string depth_indent_string(const example_predict& ec);
```

Comment:
same for these methods, they should also include `debug` or `dbg` and/or be capitalized (even though they aren't macros)

Reply:
Not sure about capitalized since that's not following convention in our code.  Adding debug.